pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -274,6 +275,19 @@ type ACMEIssuerDNS01ProviderRFC2136 struct {
 	TSIGAlgorithm string `json:"tsigAlgorithm"`
 }
 
+type ACMEIssuerDNS01ProviderWebhook struct {
```

Comment:
Could you add a comment to this struct, as well as comments on all fields? (preferably for the fields, including an example? i.e. for the URL).

For metadata it'd be great to explain when and why someone may use it 😄